src/zkevm_specs/evm/instruction.py

```diff
@@ -500,9 +500,9 @@ def tx_log_lookup(
             RW.Write,
             RWTableTag.TxLog,
             key1=tx_id,
-            key2=log_id,
+            key2=FQ(index + int(field_tag) << 32 + log_id.expr().n << 48),
             key3=FQ(field_tag),
```

Comment:
Is it possible to remove the field_tag from this key, since it already appears in key2?

Reply:
I don't see no bad effect for remaining it so far :)

Reply:
It doesn't make sense for it to appear twice. Does anything use the fact that `key3=FQ(field_tag)` rather than `key3=FQ(0)`?

Reply:
@z2trillion  sound good to me ,  Will try to remove it.

Reply:
udpated in [9187833](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/220/commits/91878339cd1b0bca2632c7af3a2d4deb2b2eb87b) thanks !